actix/src/sec-4-context.md

```diff
@@ -1 +1,140 @@
-[WIP] 
\ No newline at end of file
+# Context
+
+Actors all maintain an internal execution context, or state. This
+allows an actor to determine it's own Address, change mailbox limits,
+or stop it's execution.
+
+## Mailbox
+
+All messages go to the actor's mailbox first, then the actor's execution context
+calls specific message handlers. Mailboxes in general are bounded. The capacity is
+specific to the context implementation. For the `Context`  type the capacity is set to
+16 messages by default and can be increased with
+[*Context::set_mailbox_capacity()*](../actix/struct.Context.html#method.set_mailbox_capacity).
+
+```rust
+# extern crate actix;
+# use actix::prelude::*;
+struct MyActor;
+impl Actor for MyActor {
+    type Context = Context<Self>;
+
+    fn started(&mut self, ctx: &mut Self::Context) {
+        ctx.set_mailbox_capacity(1);
+    }
+}
+
+# fn main() {
+# System::new("test");
+let addr = MyActor.start();
+# }
+```
+
+Remember that this doesn't apply to `Addr::do_send(M)` which bypasses the Mailbox queue limit.
+
+## Getting your actors Address
```

Comment:
Add note about `notify` and `notify_later` which are methods of `AsyncContext`

Since they bypass mailbox, they are not limited by it

Reply:
`notify` and `notify_later` are methods of `AsyncContext` and therefore can only be invoked from within actor using context.

It is not part of `Addr` and while your statement is true for all three of them, `notify` and `notify_later` bypass Mailbox entirely